spec/requests/admin/court_application_hearing_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe "/admin/court_application_hearing", type: :request do
```

Comment:
Is there a reason we're doing this as a request spec rather than a feature spec? As it's describing functionality that's accessed by the web UI I'm surprised we're not testing it by having capybara click buttons in the web UI for us.

Reply:
Just following current implementation of the mock.